lib/rules/length-zero-no-unit/__tests__/index.mjs

```diff
@@ -990,3 +990,30 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRulePreludes: ['function', /^mix/] }],
```

Comment:
question

@jeddy3 was `ignore: ['at-rule-preludes']` considered?
I am asking because the name chosen could potentially make the user mistakenly pass the name of the prelude instead of an at-rule.
I know that it would be less specific but it would also be less error prone.

Reply:
Good point. I've suggested another name elsewhere in the review.